script.js

```diff
@@ -329,98 +283,19 @@ document.addEventListener("DOMContentLoaded", function () {
       analytics.reset();
 
       // Set user as logged out and update button states
-      localStorage.setItem(loggedInKey, "false");
+      localStorage.setItem(LOGGED_IN_KEY, "false");
       updateButtonStates();
     });
+}
 
-  // Update button states based on login status
-  function updateButtonStates() {
-    const loggedIn = localStorage.getItem(loggedInKey) === "true";
-    const loginButton = document.getElementById("loginButton");
-    const logoutButton = document.getElementById("logoutButton");
-
-    if (loggedIn) {
-      loginButton.setAttribute("disabled", "true");
-      logoutButton.removeAttribute("disabled");
-    } else {
-      loginButton.removeAttribute("disabled");
-      logoutButton.setAttribute("disabled", "true");
-    }
-  }
-
+document.addEventListener("DOMContentLoaded", function () {
```

Comment:
you can see this function as the kick off function to it all and an outline of each major piece of functionality